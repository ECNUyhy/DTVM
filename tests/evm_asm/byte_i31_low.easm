// BYTE(31, 0x..00CD) = 0xCD
PUSH32 0x00000000000000000000000000000000000000000000000000000000000000CD
PUSH1  0x1F
BYTE
PUSH1 0x00
MSTORE
PUSH1 0x20
PUSH1 0x00
RETURN
