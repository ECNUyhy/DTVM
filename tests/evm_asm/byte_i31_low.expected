status: success
error_code: 0
stack: []
memory: '00000000000000000000000000000000000000000000000000000000000000CD'
storage: {}
transient_storage: {}
return: '00000000000000000000000000000000000000000000000000000000000000CD'
events: []
